templates/user/settings/security/security.tmpl

```diff
@@ -1,11 +1,17 @@
 {{template "user/settings/layout_head" (dict "ctxData" . "pageClass" "user settings security")}}
+	{{if not ($.UserDisabledFeatures.Contains "manage_mfa" "manage_credentials")}}
 	<div class="user-setting-content">
+		{{if not ($.UserDisabledFeatures.Contains "manage_mfa")}}
 		{{template "user/settings/security/twofa" .}}
 		{{template "user/settings/security/webauthn" .}}
+		{{end}}
+		{{if not ($.UserDisabledFeatures.Contains "manage_credentials")}}
```

Comment:
What is the reasoning behind that? Is it to prevent them from unlinking their primary account?

Reply:
> What is the reasoning behind that? Is it to prevent them from unlinking their primary account?

I think I referred to the wrong file. This comment is for `navbar.tmpl`, and you have already added it in commit 3df7358.

And there is another bug: when I click `Blocked users`, the `Account` option will appear again, even though `manage_credentials` and `deletion` are true.

Reply:
Thanks for catching that! It should be fixed in https://github.com/go-gitea/gitea/pull/31535/commits/c9487b41eecf4a25d85a862e24b5d1aab58b8920.